gm4_teleportation_anchors/data/gm4_teleportation_anchors/functions/player/visuals_jam.mcfunction

```diff
@@ -0,0 +1,8 @@
+# @s = player that was jammed
+# run from player/find_aec
+
+particle minecraft:angry_villager ~ ~1.3 ~ 0.3 0.2 0.3 1 5
```

Comment:
Maybe use local coordinates to force this particle infront of the player? That way it is a little more visible.

Reply:
Made this change with the latest commit